staging/README.md

```diff
@@ -1,3 +1,15 @@
-This staging/src/k8s.io/client-go directory is the staging area of the client repo. It contains a versioned client, tools built around the client like the reflector, and all the client dependencies. The content will be periodically published to k8s.io/client-go repo.
-The staged content is copied from the main repo, i.e., k8s.io/kubernetes, with directory rearrangement and necessary rewritings. To sync the content with the latest code in your local k8s.io/kubernetes, you need to run `godep restore` in k8s root directory, then run staging/copy.sh.
-vendor/k8s.io/client-go is a symlink pointing to this staging area, so to use the packages in the staging area, you can import it as "vendor/client-go/<package-name>", as if the client were vendored. The client will be vendored from k8s.io/client-go for real after the test matrix is converted to vendor k8s components.
+This staging/src/k8s.io/client-go directory is the staging area of the client
```

Comment:
I attempted a generic rewrite of this and the first paragraph based on my (very) naive understanding of how this works. I'm happy to incorporate any specific suggestions from the experts or roll back this second commit as we've reached the limit of my knowledge in this area.